portfolio/src/main/java/com/google/sps/models/Comment.java

```diff
@@ -61,6 +77,7 @@ public static ValidationResponse validate(Comment comment) {
         public static final String KIND = "Comment";
```

Comment:
Going by the other capitalization, should this be "comment" instead of "Comment"?

Reply:
I have changed this to "comment".